accepted/2022/enable-512-vectors.md

```diff
@@ -0,0 +1,518 @@
+# Enable EVEX and 512-bit Vectors in RyuJIT
+
+.NET support for SIMD acceleration through its Vector APIs (`Vector<T>`, `Vector64`, `Vector128`, and `Vector256`) allow developers to harness the power of SIMD hardware acceleration without expert-level knowledge of underlying complex instruction set architectures and extensive platform-dependent intrinsics. 
+
+We propose to extend .NET support for SIMD acceleration to 512-bit vectors, where `Vector<T>` can transparently select for this wider vector width if available. In addition, we expose a new `VectorMask` API, which allows for a more declarative and programmer-friendly method for performing conditional SIMD operations.
+
+One such realization of 512-bit SIMD acceleration is the `AVX512` family of instruction sets for `X86` architecture which this proposal addresses for implementation details.
+
+**Owner** [Anthony Canino](https://github.com/anthonycanino) 
+
+## Scenarios and User Experience
+
+### Vector<T> Usage
+
+Existing code that utilizes `Vector<T>` to perform vectorized sum over a span might look something like the following snippet:
+
+```C#
+public int SumVector(ReadOnlySpan<int> source)
+{
+  Vector<int> vresult = Vector<int>.Zero;
+
+  int lastBlockIndex = source.Length - (source.Length % Vector256<int>.Count);
```

Comment:
This should be `Vector<int>.Count` since we're working with `Vector<int>`
```suggestion
  int lastBlockIndex = source.Length - (source.Length % Vector<int>.Count);
```